addon/components/bourbon-search-select-field.js

```diff
@@ -5,8 +5,9 @@ import { observer, computed } from '@ember/object';
 
 import layout from '../templates/components/bourbon-search-select-field';
 import SelectMixin from 'bourbon/mixins/select';
+import clickHandlerMixin from 'bourbon/mixins/clickHandler';
```

Comment:
🍹 probably best to be consistent with the casing
so `ClickHandlerMixin` and `'bourbon/mixins/click-handler'`
unless those don't work for some reason

Reply:
good call.  let me update that.  :)